geospaas_processing/downloaders.py

```diff
@@ -317,6 +320,11 @@ def download_dataset(self, dataset, download_directory):
                         f"Could not write the dowloaded file to {error.filename}") from error
                 else:
                     if downloaded:
+                        if self.save_path:
+                            dataset.dataseturi_set.get_or_create(
+                                dataset=dataset,
+                                uri = os.path.join(download_directory, file_name),
```

Comment:
You should store an absolute path. `download_directory` is not necessarily one.
You can use https://docs.python.org/3/library/os.path.html#os.path.realpath

Reply:
Done in the next commit